mappings/androidx-class-mapping.csv

```diff
@@ -257,7 +257,7 @@ android.arch.persistence.room.Update,androidx.room.Update
 android.arch.persistence.room.util.StringUtil,androidx.room.util.StringUtil
 android.arch.persistence.room.util.TableInfo,androidx.room.util.TableInfo
 android.arch.persistence.room.verifier.ColumnInfo,androidx.room.verifier.ColumnInfo
-android.arch.persistence.room.verifier.DatabaseVerificaitonErrors,androidx.room.verifier.DatabaseVerificaitonErrors
+android.arch.persistence.room.verifier.DatabaseVerificationErrors,androidx.room.verifier.DatabaseVerificationErrors
```

Comment:
Updated from the Google site, seems they had a typo.

Reply:
Do they do it manually?

Reply:
Yep. I downloaded it manually as well.